ArduSub/Sub.cpp

```diff
@@ -12,6 +12,9 @@
    You should have received a copy of the GNU General Public License
    along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
+
+// ArduSub scheduling, originally copied from ArduCopter
+
```

Comment:
can we move this comment down to before the SCHED_TASK define? or maybe just remove it?

Reply:
I've pushed a patch to remove it.